ethereum/executionlayer/src/main/java/tech/pegasys/teku/ethereum/executionlayer/BuilderBidValidatorImpl.java

```diff
@@ -120,21 +122,19 @@ public ExecutionPayloadHeader validateAndGetPayloadHeader(
     final UInt64 proposedGasLimit = executionPayloadHeader.getGasLimit();
 
     if (parentGasLimit.equals(preferredGasLimit) && proposedGasLimit.equals(parentGasLimit)) {
-      return executionPayloadHeader;
+      return;
```

Comment:
I don't understand checks since this line. It couldn't fail. What's in the spec?
Ahh. I'm getting this. It's just an event log if none of this matches. I'd add some comment on :118

Reply:
Thanks. Added a more clarifying comment.